### PR TITLE
Challenge: hamzamogni submission

### PR DESCRIPTION
**Pull request by hamzamogni**
probably shooting myself in the foot using python 

Note: All used libraries are form std lib of python (for threading, multiprocessing and mapped files)

### CODE REVIEW

File: submissions/hamzamogni/main.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import os
+import multiprocessing as mp
```
Comment (aboullaite) [inline submissions/hamzamogni/main.py:2]: Only std libs are allowed

**Review by aboullaite — CHANGES_REQUESTED** [review]

File: submissions/hamzamogni/main.py (hunk 1 above)
Comment (hamzamogni) [reply submissions/hamzamogni/main.py:2]: those are part of python's std lilb

https://docs.python.org/3/library/multiprocessing.html
https://docs.python.org/3/library/os.html#module-os


Thanks

Comment (hamzamogni) [reply submissions/hamzamogni/main.py:2]: I am using os, multiprocessing, mmap, collections and threading ... all part of standard library

Comment (hamzamogni) [reply submissions/hamzamogni/main.py:2]: mmap: https://docs.python.org/3/library/mmap.html#module-mmap
threading: https://docs.python.org/3/library/threading.html#module-threading
collections: https://docs.python.org/3/library/collections.html#module-collections

**Review by aboullaite — APPROVED** [review]

**Pull request merged**